rubrics/tool/src/main/java/org/sakaiproject/rubrics/controller/GeneratePdfController.java

```diff
@@ -0,0 +1,268 @@
+            @RequestHeader(value = X_LANG_HEADER) String lang, @RequestHeader(value = "toolId") String toolId,
+            @RequestHeader(value = "itemId") String entityId,
+            @RequestHeader(value = "evaluatedItemId") String evaluatedItemId)
+            throws CloneNotSupportedException {
+
+        Rubric sourceRubric = null;
+        String fileRubricName = null;
+        List<Evaluation> evals = null;
+        StreamingResponseBody stream = null;
+
+        try {
+            sourceRubric = rubricRepository.findById(Long.parseLong(sourceId))
+                .orElseGet(() -> rubricsConfiguration.getDefaultLayoutConfiguration(lang).getDefaultRubric());
+
+            evals = evaluationRepository.findByToolIdAndAssociationItemIdAndEvaluatedItemId(toolId, entityId, evaluatedItemId);
+            String fileName = sourceRubric.getTitle();
+            fileRubricName = fileName + ".pdf";
+            File tempFile = createPdf(sourceRubric, fileRubricName, evals);
+            Path pdfPath = Paths.get(tempFile.getAbsolutePath());
+            stream = outputStream -> {
+                Files.copy(pdfPath, outputStream);
+            };
+            //Files.deleteIfExists(pdfPath);
+        } catch (DocumentException | IOException ex) {
+            log.error(ex.getMessage(), ex);
+        }
+
+        HttpHeaders respHeaders = new HttpHeaders();
+        respHeaders.setContentType(new MediaType("application", "pdf"));
+        respHeaders.setContentDisposition( ContentDisposition.builder("attachment") .filename(fileRubricName).build());
+        return ResponseEntity.ok()
+                .headers(respHeaders)
+                .contentType(MediaType.parseMediaType("application/pdf"))
+                .body(stream);
+    }
+
+    public File createPdf(Rubric sourceRubric, String fileRubricName, List<Evaluation> evals)
+            throws DocumentException, IOException {
+
+        String userId = sessionManager.getCurrentSessionUserId();
+        final Locale locale = preferencesService.getLocale(userId);
+        // Count points
+        Integer points = 0;
+        if (null != evals && !evals.isEmpty()) {
+            points = evals.stream().flatMap(a -> a.getCriterionOutcomes().stream()).mapToInt(x -> x.getPoints().intValue()).sum();
+        }
+        // Create pdf document
+        Document document = new Document(PageSize.A4.rotate());
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        PdfWriter.getInstance(document, out);
+        File tempFile = new File(fileRubricName); 
+        PdfWriter.getInstance(document, new FileOutputStream(fileRubricName)); 
+        document.open();
+
+        PdfPTable table = new PdfPTable(1);
+        PdfPCell header = new PdfPCell();
+        String rubricName = sourceRubric.getTitle();
+        Paragraph paragraph = new Paragraph(messageSource.getMessage("export_rubric_title", null, locale) + rubricName +"\n", font);
```

Comment:
This is related to the UI, as you can see you're harcoding the order of the rubric name instead of building the string.